gamemodes/cinema/gamemode/modules/theater/sh_video.lua

```diff
@@ -229,6 +234,7 @@ if CLIENT then
 
                     if v and not v:ShouldTrust() then
                         v.FORCETRUST = true
+                        if CURRENTKEY then TEMP_TRUST[CURRENTKEY] = true end
```

Comment:
this is dumb, just url.parse2 the v:Key() and get the host again

Reply:
I forgot I could do that to video objects